pages/apply.html

```diff
@@ -0,0 +1,380 @@
+        </div>
+
+      </fieldset>
+
+
+      <fieldset class="resume">
+
+        <h2><legend>Resume:<span class="required">*</span></legend></h2>
+
+        <div class="usa-input">
+
+          <p>For guidance on how to build a government-style resume, please visit this link: <a href="https://pages.18f.gov/joining-18f/government-style-resume-guide/">https://pages.18f.gov/joining-18f/government-style-resume-guide/</a></p>
+
+          <p>Please copy and paste your résumé (or a link to it) below. Do NOT include your Social Security Number. Please be aware that cutting and pasting will remove most formatting.</p>
+
+          <textarea name="entry.710467749" rows="8" cols="0" class="ss-q-long" id="entry_710467749" dir="auto" aria-label="Resume: Please copy and paste your résumé (or a link to it) below. Do NOT include your Social Security Number. Please be aware that cutting and pasting will remove most formatting." aria-required="true" required></textarea>
+
+        </div>
+
+      </fieldset>
+
+
+      <fieldset class="why">
+
+        <h2><legend>About the "why:"</legend></h2>
+
+        <div class="usa-input">
+
+          <label class="ss-q-item-label" for="entry_510169774">Why do you want to join 18F?<span class="required">*</span></label>
+          <textarea name="entry.510169774" rows="8" cols="0" class="ss-q-long" id="entry_510169774" dir="auto" aria-label="Why do you want to join 18F?  " aria-required="true" required></textarea>
+
+        </div>
+
+
+        <div class="usa-input">
+
+          <label class="ss-q-item-label" for="entry_1108678713">Tell us why you&#39;re interested in the role you're applying for.<span class="required">*</span></label>
+          <textarea name="entry.1108678713" rows="8" cols="0" class="ss-q-long" id="entry_1108678713" dir="auto" aria-label="Tell us why you&#39;re interested in the role you're applying for. " aria-required="true" required></textarea>
+
+        </div>
+
+      </fieldset>
+
+
+      <input type="hidden" name="draftResponse" value="[,,&quot;-9175896065341356209&quot;]
+      ">
+      <input type="hidden" name="pageHistory" value="0">
+      <input type="hidden" name="fvv" value="0">
+      <input type="hidden" name="fbzx" value="-9175896065341356209">
+
+      <div class="button_wrapper">
+        <input type="submit" value="Submit" id="ss-submit" class="jfk-button jfk-button-action " />
+      </div>
+
+    </form>
+
+  </div>
+
+  <div id="curtain"></div>
+
+  <div id="modal" aria-hidden="false" aria-labelledby="Successful form submission" role="dialog">
+   <div class="usa-alert usa-alert-success">
+      <div class="usa-alert-body">
+        <h1 class="usa-alert-heading">Thank you!</h1>
+        <p class="usa-alert-text">Thank you for your application. We'll be in touch if there is a fit. For questions, please email us at join18f@gsa.gov.</p>
```

Comment:
I'll check whether the existing confirmation page sufficiently covers this. In the meantime, @hestherchang: is it true that the Talent Team plans to reply only if they want to proceed with an applicant? I ask because most applicants expect to hear back either way.